src/routes/tools/[slug]/+page.server.ts

```diff
@@ -1,16 +1,9 @@
 import { supabase } from '$lib/db';
-import { error as svelteError } from '@sveltejs/kit';
-
-export async function load({ params }: { params: { slug: string } }) {
-	const { data, error } = await supabase.from('tools').select('*').eq('slug', params.slug);
-
-	if (error || !data) {
-		throw svelteError(404, { message: 'Not found' });
-	}
-
-	const [tool] = data;
 
+export async function load() {
+	const { data, error } = await supabase.from('tools').select('*');
+	if (!error) return;
 	return {
-		tool
+		data
```

Comment:
Do you mind keeping it as I had it there or was there a reason to revert those changes?